src/components/ProfilePicture.js

```diff
@@ -1,19 +1,24 @@
 import { img } from 'react-hyperscript-helpers'
 import { getUser } from 'src/libs/auth'
+import * as Utils from 'src/libs/utils'
 
 
 const ProfilePicture = ({ size, style, ...props } = {}) => {
   // Note Azure logins don't currently have an imageUrl, so don't render anything.
   // See TOAZ-147 to support this in the future.
   const imageUrl = getUser().imageUrl
-  return imageUrl && img({
-    alt: 'Google profile image',
-    src: imageUrl,
-    height: size, width: size,
-    style: { borderRadius: '100%', ...style },
-    referrerPolicy: 'no-referrer',
-    ...props
-  })
+
+  return Utils.cond(
+    [!imageUrl, () => null],
+    () => img({
+      alt: 'Google profile image',
+      src: imageUrl,
+      height: size, width: size,
+      style: { borderRadius: '100%', ...style },
+      referrerPolicy: 'no-referrer',
+      ...props
+    })
+  )
 }
```

Comment:
[Nit] Including my tinkering below in case it reads more easily than `Utils.cond()` and a bit more clear about `null`ness.

```javascript
const ProfilePicture = ({ size, style, ...props } = {}) => {
  // Note Azure logins don't currently have an imageUrl, so don't render anything.
  // See TOAZ-147 to support this in the future.
  const { imageUrl = null } = getUser()

  return _.isNull(imageUrl) ?
    null :
    img({
      alt: 'Google profile image',
      src: imageUrl,
      height: size, width: size,
      style: { borderRadius: '100%', ...style },
      referrerPolicy: 'no-referrer',
      ...props
    })
}
```

Reply:
Don't know why I didn't think of this before, but there's a simpler fix: https://github.com/DataBiosphere/terra-ui/pull/3408/commits/7c9d25195b0e565223e03de5ba04db79d4b8e5b7.

Reply:
I didn't know [booleans are ignored](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored) in this case. Some seem to favour returning `null` for explicitness but  🤷🏼